### PR TITLE
{chem}[foss/2022a] Cantera v2.6.0 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by iotaka**
CVodes from SUNDIALS 6.3.0 was bugged, with 6.5.1 seems working fine

```
CanteraError thrown by CVodesIntegrator::integrate:
CVodes error encountered. Error code: -4
```

In https://github.com/LLNL/sundials/releases/tag/v6.5.0
was solved: Fixed an underflow bug during root finding in ARKODE, CVODE, CVODES, IDA and IDAS.

### CODE REVIEW

File: easybuild/easyconfigs/c/Cantera/Cantera-2.6.0-foss-2022a.eb [hunk 1]
```diff
@@ -15,9 +15,13 @@ checksums = ['7273622ea76a53373cee820f939613b15eea3dd83db6e1b127c5ed043f77dc5b']
 
 dependencies = [
     ('Python', '3.10.4'),
-    ('SciPy-bundle', '2022.05'),
```
Comment (casparvl) [inline easybuild/easyconfigs/c/Cantera/Cantera-2.6.0-foss-2022a.eb:18]: From your PR commit message, it was clear why the `SUNDIALS` version changed. Can you comment on why `SciPy-bundle` was removed as dependency, and why `h5py`-`PyQt5` were added?

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in total)
tcn305.local.snellius.surf.nl - Linux Rocky Linux 8.7, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/fb8db5baf1b9e20f6c006a61429f59ca for a full test report.

**Comment by casparvl** [comment]
@boegelbot please test @ jsc-zen2

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
tcn305.local.snellius.surf.nl - Linux Rocky Linux 8.7, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/c39162e0fa7caced7760371e929fac7d for a full test report.

File: easybuild/easyconfigs/c/Cantera/Cantera-2.6.0-foss-2022a.eb (hunk 1 above)
Comment (iotaka) [reply easybuild/easyconfigs/c/Cantera/Cantera-2.6.0-foss-2022a.eb:18]: https://www.cantera.org/compiling/installation-reqs.html#sec-installation-reqs

H5 is an optional dependency requested by final users. SciPy-bundle still comes from dependency . PyQt5 for plot charts `matplotlib.use('QtAgg')` for instance.

Comment (casparvl) [reply easybuild/easyconfigs/c/Cantera/Cantera-2.6.0-foss-2022a.eb:18]: > H5 is an optional dependency requested by final users.

Makes sense, it add optional functionality. That's great.

> SciPy-bundle still comes from dependency

Normally I would say "If X is a dependency of Y, it should be listed, even if X is also a dependency of Z, which is a dependency of Y". However, from your link, I see that `scipy` is optional, and only needed for some examples. So, removing it is fine by me.

> PyQt5 for plot charts matplotlib.use('QtAgg') for instance

Well, according to that argument, it should be a dependency for `matplotlib` :) (which it also isn't, but that's another discussion), but not for Cantera. If users want to use the `QtAgg` backend, they can load the `PyQt5` module themselves. I guess this is one of the ambiguous cases of how to deal with optional dependencies in EasyBuild. One reason to be a bit conservative here is that Qt5 is big, takes ages to build, and build problems are quite common. Thus, you don't really want to include it as a hard dependency unless really needed. By not including it as a dependency here (or in matplotlib), Cantera (and matplotlib) can be installed and used, even on systems where building Qt5 proves problematic. Summarizing: please remove it from the dependency list :)

Comment (iotaka) [reply easybuild/easyconfigs/c/Cantera/Cantera-2.6.0-foss-2022a.eb:18]: I removed PyQt5 as dependencies. Tnx for your support :)

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/c/Cantera/Cantera-2.6.0-foss-2022a.eb (hunk 1 above)
Comment (casparvl) [reply easybuild/easyconfigs/c/Cantera/Cantera-2.6.0-foss-2022a.eb:18]: Very welcome :)

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
tcn331.local.snellius.surf.nl - Linux Rocky Linux 8.7, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/157f344e192fef204dee7698e1c02a93 for a full test report.

**Review by casparvl — APPROVED** [review]
Lgtm!

**Comment by casparvl** [comment]
Going in, thanks @iotaka!

**Pull request merged**